README.md

```diff
@@ -7,14 +7,15 @@
 
 A transform stream used to censor data from objects before passing them down the pipeline.
 
-## `new WhiteOut (filter, [options])`
+## `new WhiteOut (filter, [root], [options])`
 
 Creates a new `WhiteOut` transform stream with the following arguments.
 - `filter` - a key value pair where `key` is the property value to censor on the object payload and the `value` is the censor type.
   - Valid options for `value` are "censor", "remove" and a RegExp.
     - "censor" - replaces the value with a string of "X"s.
     - "remove" - completely removes the key from object
     - Anything else will be treated as a `RegExp` definition and will be passed into `new RegExp`.
+- `[root]` - an object string path (ex `'response.payload'`) that will be used when the censor algorithm starts. Useful for censoring only a subsection of the entire `data` object. Defaults to `undefined` which means the entire `data` object will be traversed. For performance reasons, it is recodmended to set `root` to only the specific segment of `data` you wish to filter.
```

Comment:
Hm, I never noticed before. Just pointing it out. Feel free to address it or not.